policy/release/attestation_task_bundle.rego

```diff
@@ -106,7 +106,7 @@ warn contains result if {
 #   For each Task in the SLSA Provenance attestation, check if the Tekton Bundle used is
 #   a trusted task.
 # custom:
-#   short_name: task_ref_bundles_acceptable
+#   short_name: task_ref_bundles_trusted
```

Comment:
I know of at least one user that excludes `attestation_task_bundle.task_ref_bundles_acceptable`. We could simply add the new name now to the policies in [here](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/tree/main/config/OP/EnterpriseContractPolicy).